mvc.md

```diff
@@ -0,0 +1,123 @@
+# Psuedo Code
+
+
+## Model
+- Update the model fields to match the front end requirements.
+- Does education have any functions?  No, only for ReportModel. those functions talk directly to database AND/OR helpers
+
+doesn't have a user field...once they sign in
+
```

Comment:
Thanks for the notes here, helps to visualize the dev process

Reply:
Awesome, in the next PR we will absolutely release better notes, especially the data structure. Some updates occurred as we began coding and realized we needed another Model